gfx/UltimateCataclysm/pngs_human_body_32x36/overlay/wielded/by_Guake/overlay_wielded_black_box.json

```diff
@@ -0,0 +1,4 @@
+{
+    "id": [ "overlay_wielded_black_box" ],
```

Comment:
unless `overlay_wielded_black_box` is also used in json somewhere else this is unecessary. Naming the file with the correct id is enough if you want that sprite to be used just for this id